samples/YesSql.Samples.Hi/Program.cs

```diff
@@ -9,18 +10,18 @@ namespace YesSql.Samples.Hi
 {
     internal class Program
     {
-        static void Main(string[] args)
+        public static async Task Main(string[] args)
         {
-            MainAsync(args).GetAwaiter().GetResult();
-        }
+            var filename = "yessql.db";
 
-        static async Task MainAsync(string[] args)
-        {
-            var store = await StoreFactory.CreateAndInitializeAsync(
-                new Configuration()
-                    .UseSqlServer(@"Data Source =.; Initial Catalog = yessql; Integrated Security = True")
```

Comment:
was quite useful that this sample was using Sql Server when we come to test things on it.

Reply:
Then shall I move the showcase to a different sample, if that's ok? Trying to avoid running SQL server for this.